docs/tutorials/sigstore-keyless.md

```diff
@@ -226,10 +226,10 @@ freetsa_root_b64="$(echo "$freetsa_root" | openssl base64 -A)"
 cp policy-template.json policy.json
 
 # Use double quotes around variables in sed commands to preserve newlines
-sed -i '' "s|{{FULCIO_ROOT}}|$fulcio_root_b64|g" policy.json
-sed -i '' "s|{{FULCIO_INT}}|$fulcio_int_b64|g" policy.json
-sed -i '' "s|{{FREETSA_ROOT}}|$freetsa_root_b64|g" policy.json
-sed -i '' "s|{{EMAIL}}|$email|g" policy.json
+sed -i "s|{{FULCIO_ROOT}}|$fulcio_root_b64|g" policy.json
```

Comment:
Without the `''` I'm getting this error on my Apple silicon Mac:
```
sed: 1: "policy.json": extra characters at the end of p command
```
What OS and shell program are you using?